tests/urls.py

```diff
@@ -0,0 +1,3 @@
+from django.urls import include, path
+
+urlpatterns = [path("icl/", include("imperial_coldfront_plugin.urls"))]
```

Comment:
I don't quite follow here: what's the reason for redirecting through `tests.urls`?

Reply:
Oof. Hard to debug automagicness, then... :disappointed: